images/docker-dind/Dockerfile

```diff
@@ -27,10 +27,8 @@ RUN curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID"
     "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
     $(lsb_release -cs) stable"
 
-ENV DOCKER_VERSION=18.06.1*
-
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends docker-ce=${DOCKER_VERSION} && \
+    apt-get install -y --no-install-recommends docker-ce && \
```

Comment:
I would prefer to keep the docker version explicit so that we can 1) track down changes and 2) easily attach an update workflow for this image using Prow jobs.

Reply:
I agree with @zuc

Reply:
Makes sense!